components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -118,17 +118,18 @@ type Config struct {
 	Gardener    gardener.Config
 	Kubeconfig  kubeconfig.Config
 
-	KymaVersion                                string
-	EnableOnDemandVersion                      bool `envconfig:"default=false"`
-	ManagedRuntimeComponentsYAMLFilePath       string
-	NewAdditionalRuntimeComponentsYAMLFilePath string
-	SkrOidcDefaultValuesYAMLFilePath           string
-	SkrDnsProvidersValuesYAMLFilePath          string
-	DefaultRequestRegion                       string `envconfig:"default=cf-eu10"`
-	UpdateProcessingEnabled                    bool   `envconfig:"default=false"`
-	UpdateSubAccountMovementEnabled            bool   `envconfig:"default=false"`
-	LifecycleManagerIntegrationDisabled        bool   `envconfig:"default=true"`
-	ReconcilerIntegrationDisabled              bool   `envconfig:"default=false"`
+	KymaVersion                                                         string
+	EnableOnDemandVersion                                               bool `envconfig:"default=false"`
+	ManagedRuntimeComponentsYAMLFilePath                                string
+	NewAdditionalRuntimeComponentsYAMLFilePath                          string
+	SkrOidcDefaultValuesYAMLFilePath                                    string
+	SkrDnsProvidersValuesYAMLFilePath                                   string
+	DefaultRequestRegion                                                string `envconfig:"default=cf-eu10"`
+	UpdateProcessingEnabled                                             bool   `envconfig:"default=false"`
+	UpdateSubAccountMovementEnabled                                     bool   `envconfig:"default=false"`
+	LifecycleManagerIntegrationDisabled                                 bool   `envconfig:"default=true"`
+	ReconcilerIntegrationDisabled                                       bool   `envconfig:"default=false"`
+	AvsMaintenanceModeDuringUpgradeAlwaysDisabledGlobalAccountsFilePath string
```

Comment:
"long property name" competition won!

Reply:
I knew you'll like it :) The name reflects env var name associated with deployment which is connected to the value in chart's values.yaml. It's just a temporary variable, we will remove it when zero downtime maintenance is default.